# ChromHL


external link: https://github.com/TeifLab/ChromHL

 Developed by Graeme Thorn and Vladimir Teif
 
## Description

The mammalian epigenome contains thousands of heterochromatin nanodomains (HNDs) marked by di- and trimethylation of histone H3 at lysine 9, which have a typical size of 3-10 nucleosomes. However, the (epi)genetic determinants of their location and boundaries are only partly understood. Here, we compare four HND types in mouse embryonic stem cells, that are defined by histone methylases SUV39H1/2 or GLP, transcription factor ADNP or chromatin remodeller ATRX. Based on a novel chromatin hierarchical lattice framework termed ChromHL, we are able to predict HND maps with singe-nucleotide resolution. We find that HND nucleation can be rationalized by DNA sequence specific protein binding to PAX3/9, ADNP and LINE1 repeats. Depending on type of microdomains, boundaries are determined either by CTCF binding sites or by nucleosome-nucleosome and nucleosome-HP1 interactions. Our new framework allows predicting how patterns of H3K9me2/3 and other chromatin nanodomains are established and changed in processes such as cell differentiation. 

## How to cite
Thorn G.J., Clarkson C.T., Rademacher A., Mamayusupova H., Schotta G., Rippe K., Teif V.B. (2020) DNA sequence-dependent formation of heterochromatin nanodomains. bioRxiv 2020.12.20.423673 https://www.biorxiv.org/content/10.1101/2020.12.20.423673v1
